CMakeLists.txt

```diff
@@ -549,6 +550,7 @@ endif()
 
 if(${BUILD_PYTHON_WRAPPING})
     set(required_python_version 3)
+    set(Python3_ROOT_DIR ""  CACHE PATH "Top level directory containing Python3 ")
```

Comment:
I'm wondering if we could help users with a slightly more descriptive `docstring`. Perhaps something like "Top level directory containing Python3. For conda environments, this can also be set to the top level directory for a specific environment."